extension/src/popup/components/manageAssets/AddToken/index.tsx

```diff
@@ -106,6 +107,9 @@ export const AddToken = () => {
   const [isSearching, setIsSearching] = useState(false);
   const [hasNoResults, setHasNoResults] = useState(false);
   const [isVerifiedToken, setIsVerifiedToken] = useState(false);
+  const [isVerificationInfoShowing, setIsVerificationInfoShowing] = useState(
```

Comment:
e2e tests for this whole flow will be added in the manifest v3 testing issue: https://stellarorg.atlassian.net/browse/WAL-1375